tests/unit/test_messagepack.py

```diff
@@ -112,21 +112,30 @@ def v1_packer():
         ),
     ],
 )
-def test_pack_and_unpack_v1(v1_packer, message_class, init_args, expect_values):
+@pytest.mark.parametrize("protocol_version", [None, 1])
+def test_pack_and_unpack(message_class, init_args, expect_values, protocol_version):
+    if protocol_version is None:
+        do_pack = pack
+        do_unpack = unpack
+    else:
+        packer = MessagePacker(default_protocol_version=protocol_version)
+        do_pack = packer.pack
+        do_unpack = packer.unpack
+
     if expect_values is None:
         expect_values = init_args
 
     message_obj = message_class(**init_args)
 
-    on_wire = v1_packer.pack(message_obj)
+    on_wire = do_pack(message_obj)
     # first byte (version byte) "1"
     assert on_wire[0:1] == b"\x01"
```

Comment:
Should the version byte test against the value of `protocol_version` instead of hardcoding to `b"\x01"`?